nominees/xr-accessibility-project.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "XR Accessibility Project",
+  "aliases": [
+    "XRAP"
+  ],
+  "description": "An index of resources for people creating accessible XR (extended reality) applications.",
+  "website": "https://github.com/XRAccessibility/xraccessibility.github.io",
+  "license": [
+    {
+      "spdx": "CC-BY-SA-4.0",
+      "licenseURL": "https://github.com/XRAccessibility/xraccessibility.github.io/blob/main/README.md"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 9,
+      "evidenceText": "Supports accessible communication, training, and education XR infrastructure so that people with disabilities can benefit from the new applications of these technologies."
+    },
```

Comment:
This is good. Is it asking too much to tweak this wording around some the SDG Targets, maybe 9.b and 9.c?

![Screenshot of Goal 9 page from the U.N. S.D.G. website. Shown are Targets 9.b ("Support domestic technology development, research and innovation in developing countries, including by ensuring a conducive policy environment for, inter alia, industrial diversification and value addition to commodities") and 9.c ("Significantly increase access to information and communications technology and strive to provide universal and affordable access to the Internet in least developed countries by 2020").](https://user-images.githubusercontent.com/4721034/176592117-5d553472-fa9a-46ac-a51d-975c2ad1c7e3.png "Screenshot of Goal 9 page from the U.N. S.D.G. website. Shown are Targets 9.b (\"Support domestic technology development, research and innovation in developing countries, including by ensuring a conducive policy environment for, inter alia, industrial diversification and value addition to commodities\") and 9.c (\"Significantly increase access to information and communications technology and strive to provide universal and affordable access to the Internet in least developed countries by 2020\").")

Reply:
Resolved in commit e62b5c0